src/P5Functions/touchmouse.js

```diff
@@ -315,6 +315,8 @@ module.exports = function withTouch(p, curElement, attachEventHandler, document,
 
     if (delta && typeof p.mouseScrolled === 'function') {
       p.mouseScrolled();
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
this is actually the right place: putting them higher means that scroll can get killed off even if the sketch does not have a scroll handler, which would lead to something like a user scrolling through a page with the sketch, their cursor hitting the sketch during a page scroll operation, and the scroll suddenly getting killed off.